beangulp/importer.py

```diff
@@ -128,3 +128,55 @@ def file_date(self, file) -> Optional[date]:
           (If no date is returned, the file creation time is used. This is the
           default.)
         """
+
+
+class Importer:
```

Comment:
I love it, but I would put it at the top of the module, transfer the comments over to this one, make the comments on the previous version just say "See Importer.XXX". We want to emphasize that this is the current one.